internal/resources/grafana/resource_dashboard_permission.go

```diff
@@ -50,6 +51,12 @@ func ResourceDashboardPermission() *schema.Resource {
 				Type:        schema.TypeSet,
 				Required:    true,
 				Description: "The permission items to add/update. Items that are omitted from the list will be removed.",
+				// Ignore the org ID of the team when hashing. It works with or without it.
```

Comment:
If it works with or without it, why ignoring it then? Not sure I follow 🤔

Reply:
This could be explained better in the comment. The reason why the org ID can be ignored is that resources that refer to each other have to be in the same org anyways (except global resources like a user). So if you specify a team that is not within the same org as the permissions, the API will fail.

And the reason why it HAS to be ignored with the set function is that `1:1` (AKA: team 1 in org 1) is functionally equivalent to team `1` (that is `team 1` in an unspecified org). Lots of this is due to migration, and can definitely be improved once we deprecate the global org setting. At that point, all of the team IDs should always have the org ID in them and we can stop doing weird exceptions like this